persistence/3.0/_index.md

```diff
@@ -1,24 +1,24 @@
 ---
 title: "Jakarta Persistence 3.0 (under development)"
-date: 2020-01-15
+date: 2020-06-05
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Persistence defines a standard for management of persistence
 and object/relational mapping in Java(R) environments.
 
-* [Jakarta Persistence 3.0 Release Record]()
+* [Jakarta Persistence 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.jpa/releases/3.0-jakarta-persistence-api)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Persistence 3.0 Specification Document]() (PDF)
-* [Jakarta Persistence 3.0 Specification Document]() (HTML)
+* [Jakarta Persistence 3.0 Specification Document](./persistence_3.0-RC2.pdf) (PDF)
+* [Jakarta Persistence 3.0 Specification Document](./persistence_3.0-RC2.html) (HTML)
 * [Jakarta Persistence 3.0 Javadoc](./apidocs)
 * [Jakarta Persistence 3.0 TCK]()([sig](),[sha](),[pub]())
 * Maven coordinates
-  * [jakarta.persistence:jakarta.persistence-api:jar:3.0.0]()
+  * [jakarta.persistence:jakarta.persistence-api:jar:3.0.0-RC2](https://search.maven.org/artifact/jakarta.persistence/jakarta.persistence-api/3.0.0-RC2/jar)
 
 
 # Compatible Implementations
 
-* [Jakarta Persistence]()
+* [EclipseLink 3.0.0-M2](https://www.eclipse.org/eclipselink)
```

Comment:
Should this point to somewhere more specific?  For example, I can't find an M2 driver for EclipseLink via the link you provided.  I went to Downloads and couldn't find it.  So, I tried Milestone downloads and still didn't find it.  I then went to the 3.0 web page (https://www.eclipse.org/eclipselink/releases/3.0.php) and found a link to Download 3.0, but that takes me back this page, which only has 2.7.7 drivers on it (https://www.eclipse.org/eclipselink/downloads/index.php#27).

Reply:
idea was to create PRs and update web for each project as we get close to the GF M1 date. But by not updating the web one can save some time, so links were updated to what we have right now and will be updated eventually if new milestones become available